src/app/app.component.ts

```diff
@@ -1,10 +1,81 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit  } from '@angular/core';
+import { discardPeriodicTasks } from '@angular/core/testing';
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+import{SearchFilterPipe} from './filter.pipe'
 @Component({
   selector: 'app-root',
+
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+
 })
-export class AppComponent {
-  title = 'app';
+export class AppComponent implements OnInit{
+  tasks: Task[] = [];
+  newTask = "  ";
+  id = 1;
+  itemFilter: any = { text: '' };
+  tasksDone = [];
+   title:String;
+constructor() { 
+  this.title = " ";
+  this.tasks = []
+}
```

Comment:
Po co przypisujesz te rzeczy w konstruktorze? Po pierwsze możesz to zrobić przy deklarowaniu ich po drugie inicjalizowania danych nie robi się w konstruktorze tylko w `ngOnInit`. Po trzecie wcięcia.

Reply:
https://github.com/jedi800/Checklistaa/pull/1/commits/7d3dc5a57e118e29212c1f830130cb41a673d92c